tiddlers/stylesheet.tid

```diff
@@ -1,6 +1,5 @@
 title: $:/plugins/tgrosinger/tw5-checklist/stylesheet
 tags: $:/tags/Stylesheet
-type: text/css
```

Comment:
This type does not allow wikitext interpretation, which is used for a) transclusion of variables and b) checking the strikethrough-setting. (No type ist the same as text/vnd.tiddlywiki I believe.)